source/eitri/requirements.txt

```diff
@@ -7,3 +7,4 @@ GeoAlchemy2==0.2.4
 SQLAlchemy==1.3.3
 alembic==0.6.7
 requests==2.20.0
+typing
```

Comment:
Don't we have to specify a version here too ?

Reply:
You're right, I will do it.